addons/account/controllers/portal.py

```diff
@@ -21,6 +21,7 @@ def _prepare_portal_layout_values(self):
         ])
 
         values['invoice_count'] = invoice_count
+        values['is_documents'] = bool(invoice_count)
```

Comment:
Remove the is_documents key.

Reply:
why ? show message, if there is no documents into the account

Reply:
Cfr task. We won't show a message to tell there is nothing to show. Keep portal simple.